spec/features/events/new_spec.rb

```diff
@@ -26,11 +26,6 @@
     expect(page).to have_content(@movie_title)
   end
 
-  it 'should have a field for a name' do
-
-    expect(page).to have_field('name')
-  end
-
```

Comment:
@Trevor-Robinson why did you remove this?

Reply:
For the display of the viewing parties, it is only suppose to show the name of the movie, and not a separate title. As far as I could tell, the seperate event name field wasn't going to be used anywhere. Probably should have checked with you both before just removing though. Sorry about that.

Reply:
Oh okay! No worries at all! I was just curious